hydra-node/test/Hydra/OptionsSpec.hs

```diff
@@ -26,6 +29,21 @@ spec = parallel $
   describe "Hydra Node RunOptions" $ do
     -- NOTE: --node-id flag needs to be set so we set a default here
     let setFlags a = ["--node-id", "node-id-1"] <> a
+    it ("validateRunOptions: using more than " <> show (maximumNumberOfParties + 1) <> " parties should error out") $ do
+      let simulatedKeys = replicate 10 ['a' .. 'z']
+          cardanoKeys = take 6 simulatedKeys
+          hydraKeys = take 5 simulatedKeys
```

Comment:
Should: define these tests in term of the `maximumNumberOfParties`. For example `cardanoKeys = take (maximumNumberOfParties + 2) simulatedKeys`.